README.md

```diff
@@ -50,7 +50,17 @@ Example for local development on the host **http://localhost:3000** and staging
 
 1.  Get the auth0 application values: **DOMAIN, CLIENT_ID, CLIENT_SECRET**
 
-1.  Create file **.env.local** and copy commented-out `AUTH0_*` variables from **.env**.
+1.  Create file **.env.local** with the following contents:
+
+    ```ini
+    AUTH0_BASE_URL=http://localhost:3000
```

Comment:
Should we hardcode the localhost inside readme file?

Reply:
Good spot! I thought this section was focused on local dev, in which case we’d need `localhost:3000`. But it’s about Auth0 in general. I’ll tweak README in a follow-up PR and link it here 👍